src/coreclr/jit/emit.h

```diff
@@ -1650,10 +1649,33 @@ class emitter
 #endif                                     // MULTIREG_HAS_SECOND_GC_RET
     };
 
+    // TODO-Cleanup: Uses of stack-allocated instrDescs should be refactored to be unnecessary.
+    template <typename T>
+    struct inlineInstrDesc
+    {
+    private:
+        instrDescDebugInfo* idDebugInfo;
+        alignas(alignof(T)) char idStorage[sizeof(T)];
```

Comment:
Given the `static_assert_no_msg` below, isn't this `alignas` not useful? Or equivalent to `alignas(sizeof(instrDescDebugInfo*))`?

Reply:
In theory, some future `instrDesc` could have larger-than-pointer alignment requirement. That would certainly require some work to support and I just want it to be noisy that this is a place that would need to be fixed.